src/Sleep.js

```diff
@@ -77,10 +77,27 @@ class Sleep {
   }
 
   getGoodSleepers(date) {
-    this.data.forEach(sleep => {
-      let sleepData = this.getUserSleepData(sleep.userID);
+    let goodSleepers = [];
+    // console.log(this.data)
+    let idList = this.data.map(sleep => sleep.userID).reduce((acc, idNumber)=> {
+      if (!acc.includes(idNumber)) acc.push(idNumber);
```

Comment:
Does this line look off to you?  (potentially GitHub thing?)

Reply:
What I see is the accumulator with the bang, and we push the idNumber right after without {.  I think it's probably how it is displaying here.